canonical-web-and-design/ubuntu.com
# HTTPS everywhere

## Done

- HTTPS everywhere where appropriate
 - Update references to bare domain

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Sanity check code
